_posts/2022-09-30-chapter-tech-leads.markdown

```diff
@@ -0,0 +1,87 @@
+- Teams have different sizes depending on their needs. We have teams with only 1 developer, so we had to provide a home base for these people.
+- The future need for skills in a team depends on its product evolvement, but in the matrix, there is a natural disconnect between the product team and department. So, we more often reacted to a new demand instead of being prepared.
+This is why the Software Development department recently switched its structure.
+Previously, we had 2 teams covering 20 people each that supposedly focussed on different topics trying to avoid the caveats described above.
+This resulted in 2 team leads giving their best in keeping the balls in the air but eventually losing focus on what the company needed.
+
+Therefore, we decided to give up the team structure and form a chapter structure (as explained in the (in-)famous Spotify model) around Technical Leads as Chapter Leads to structure the How for the product teams more efficiently.
+
+![Differences between Chapter Leads and Product Owners]({{site.baseurl}}/assets/img/220930-chapter/how_what.png)
+*Difference between Chapter Lead and Product Owner ([Henrik Kniberg & Anders Ivarsson, Oct 2012](https://blog.crisp.se/wp-content/uploads/2012/11/SpotifyScaling.pdf))*
+
+## What's a Chapter & Chapter Lead for us?
+"The Chapter's more of a regular's table for its members, a place where they can touch base with like-minded and can discuss technical things. The Chapter also defines its guidelines regarding work processes (documentation, coding, etc.)."
+
+We define a chapter as a group of people that
+- shares similar expertise (or is interested in it) and
+- shares knowledge, experience, and best practices internally and with other chapters and
+- is led by a tech lead and
+- has a size of max. 10 regular employees
+
+There are many definitions for what a Chapter Lead (CL) is and what is required to become one. For us, we defined it as a senior developer who can 
+- create a vision for technical topics,
+- mentor other developers, 
+- pass the passion for development to the members, and 
+- organizes joint efforts for education and training. 
+
+For that, the CL is free to organize the Chapter and communication as seems necessary.<br/>
+ He (there are only male leads in our case at the moment) would be responsible for 
+ - creating the vision for the Chapter
+ - setting up and maintaining  the Chapter Backlog
+ - for supporting team members in becoming experts in the Chapter's mission topic.
+
+A Chapter Lead would not decide on salary rises, approve vacation days or hire people. But a CL will always give valuable input and information regarding these topics to the line manager.
+
+## How did we do the setup?
+
+### 1. Cut the elephant into pieces
+Once the goal was decided, we had to figure out how to get there, and the first step was to cut out the chapters.
```

Comment:
cut out => either "carve out" or "define"

Reply:
yes, but I wanted to put a reference to the "cut the elephant" - let's keep it